src/main/java/uk/gov/companieshouse/pscstatement/delta/mapper/StatementMapper.java

```diff
@@ -18,6 +18,7 @@
 @Mapper(componentModel = "spring")
 public interface StatementMapper {
 
+    MapperUtils mapperUtils = new MapperUtils();
```

Comment:
Can this be autowired in? The mapper is defined as being part of spring so I thought it would work.